mloop/visualizations.py

```diff
@@ -226,24 +235,33 @@ def plot_parameters_vs_cost(self):
         '''
         Create a plot of the parameters versus run number.
         '''
-        if self.num_in_costs != self.num_out_params:
-            self.log.warning('Unable to plot parameters vs costs, unequal number. in_costs:' + repr(self.num_in_costs) + ' out_params:' + repr(self.num_out_params))
-            return
+        # Only plot points for which a cost was actually measured. This may not
+        # be the case for all parameter sets if the optimization is still in
+        # progress, or ended by a keyboard interupt, etc..
+        in_costs = self.in_costs[:self.num_in_costs]
+        in_uncers = self.in_uncers[:self.num_in_costs]
+        
+        # if self.num_in_costs != self.num_out_params:
```

Comment:
Just delete this block now

Reply:
I think the specific line numbers in the above excerpt are a bit off, but I assume you wanted the commented-out code removed? If so that's now resolved by commit 821a052 below.